packages/core/src/helpers/insertBreak/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { NodeFunction } from '../../config';
+import { helpers } from '../';
+
+/**
+ * Inherit the type of the current Element when you press enter.
+ */
+export const copyInsertBreak: NodeFunction = (editor, entry) => {
+  const [currentNode] = entry;
```

Comment:
nits:
entryから渡されるものが現在であることは明示だと思うので、他のように
```suggestion
  const [node] = entry;
```
っていう命名でいいと思います！